tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "declaration": true,
   "compilerOptions": {
     "target": "ES6",
+    "skipLibCheck": true,
```

Comment:
Is this needed since we now import bootstrap types?

Reply:
Yes, otherwise TypeScript goes into the Bootstrap types and gets yelly about redefined types (or something similar; I forget the exact error message).